### ISSUE TITLE
installation of phcpy in anaconda

### ISSUE THREAD

**Issue by berceanu** [janverschelde/PHCpack#8]
Are there any PHCpy packages for pip or conda? If not, are there any plans for this in the near future?

**Comment by janverschelde** [comment]
Yes, a pip wheel distribution for Mac OS X and Linux is planned before the end of the summer.

**Comment by Dirivian** [comment]
Has this been done?

**Comment by janverschelde** [comment]
No, feel free to contribute.

**Comment by janverschelde** [comment]
Once libPHCpack has been build and placed into the phcpy source folder, phcpy can be installed with pip.
